mail-client/alpine/alpine-2.21.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools toolchain-funcs
+
+DESCRIPTION="alpine is an easy to use text-based based mail and news client"
+HOMEPAGE="http://www.washington.edu/alpine/ http://alpine.freeiz.com/alpine/ http://repo.or.cz/alpine.git/"
+SRC_URI="http://alpine.freeiz.com/alpine/release/src/${P}.tar.xz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc ipv6 kerberos ldap libressl nls onlyalpine passfile smime spell ssl threads"
+
+DEPEND="virtual/pam
+	>=sys-libs/ncurses-5.1:0=
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	ldap? ( net-nds/openldap )
+	kerberos? ( app-crypt/mit-krb5 )
+	spell? ( app-text/aspell )
+"
+RDEPEND="${DEPEND}
+	app-misc/mime-types
+	!onlyalpine? ( !mail-client/pine )
+	!<=net-mail/uw-imap-2004g
+"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	MYCONF=(
+		"--without-tcl"
+		"--with-system-pinerc=${EPREFIX}/etc/pine.conf"
+		"--with-system-fixed-pinerc=${EPREFIX}/etc/pine.conf.fixed"
+	)
+
+	if use ssl; then
+		MYCONF+=(
+			"--with-ssl"
+			"--with-ssl-include-dir=${EPREFIX}/usr/include/openssl"
+			"--with-ssl-lib-dir=${EPREFIX}/usr/$(get_libdir)"
+			"--with-ssl-certs-dir=${EPREFIX}/etc/ssl/certs"
+		)
+	else
+		MYCONF+="--without-ssl"
```

Comment:
remove this and the `"--with-ssl"` in the previous block and add `$(use_with ssl)` back to `econf` call